test/functional/feature_fastprune.py

```
@@ -0,0 +1,42 @@
+    assert_equal
+)
+from test_framework.blocktools import (
+    create_block,
+    create_coinbase,
+    add_witness_commitment
+)
+from test_framework.wallet import MiniWallet
+
+
+class FeatureFastpruneTest(BitcoinTestFramework):
+    def set_test_params(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `set_test_params`

Add return type annotation: `None`

(ANN201)